ydb/core/tx/schemeshard/schemeshard_impl.cpp

```diff
@@ -6215,6 +6255,8 @@ void TSchemeShard::Handle(TEvSchemeShard::TEvNotifyTxCompletionResult::TPtr& ev,
         executed = true;
     }
 
+    HandleBackgroundCleaningCompletionResult(ev->Get()->Record.GetTxId());
```

Comment:
Inconsistent handling -- any BackgroundCleaningCompletion will result in warning `no able to determine destination ...`.
`BackgroundCleaningTxs` should be checked here the same way as other `TxIdTo...` mappings -- explicitly and with `executed` flag setting.